.env

```
@@ -1,3 +1,11 @@
+# Postgres settings, used for local development, could be anything as long as they are consistent
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_DB=
+DB_URL=
```

I guess the DB_URL could be pre-specified.
```suggestion
POSTGRES_USER=
POSTGRES_PASSWORD=
POSTGRES_DB=
POSTGRES_HOST=localhost
POSTGRES_PORT=5432

DB_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
```

---

Ohh I was wrong :"D
We cannot use .env varibale into another .env variable.

---

Y its not possible like this. Also the .env file is under VC and I didn't want CI deployment to be effected hence the blank values.